services/todo-legacy/datasources.ts

```diff
@@ -0,0 +1,57 @@
+export const datasources: DataSourceConfig = {
+  ds: {
+    name: 'ds',
+    connector: 'mysql',
+    host: 'localhost',
+    port: 3306,
+    database: 'testdb',
+    password: 'pass',
+    user: 'root',
+  }
+ };
+
+export class DataSourceConfig {
```

Comment:
I think this should be `interface`?

Also, can we eventually move type definition into `@loopback/repository` please? It's fine to do that later, after this PR is landed.